src/server/api-server.js

```diff
@@ -77,7 +78,7 @@ class APIServer extends EventEmitter {
   startListening() {
     return new Promise((resolve, reject) => {
       this._expressServer = this._expressApp.listen(
-        this._port, this._host, () => {
+        this._port, this._serveHost, () => {
```

Comment:
The `_host` or rather `getUrl` method where it is used is referenced in a number of other places to add a security exception (https://github.com/GoogleChrome/web-push-testing-service/blob/dfb5ad6b7a0684ab5600aebc16b0498500fc8c8e/src/index.js#L276 and https://github.com/GoogleChrome/web-push-testing-service/blob/dfb5ad6b7a0684ab5600aebc16b0498500fc8c8e/src/index.js#L315). This was the reason to introduce another variable.